lib/helpers/clone.js

```diff
@@ -145,4 +143,11 @@ function cloneArray(arr, options) {
   }
 
   return ret;
-}
\ No newline at end of file
+}
+
+function cloneRegExp(regexp) {
+  const ret = new RegExp(regexp);
```

Comment:
Looking at the original [regexp-clone](https://github.com/aheckmann/regexp-clone/blob/c44446929998bef515e15e4de4e73204c1a6b1f4/index.js#L14) code, we're copying the flags and adding them as a second argument for:
`new RegExp(originalRegexp, flags);`.

Do we not need to do that?

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp/RegExp#parameters

Do we need to support less than ES2015? But I guess we can do it on the safe side.

Reply:
```suggestion
  const ret = new RegExp(regexp.source, regexp.flags);
```